### PR TITLE
Comments router, add reply, get comments/replies

### PR DESCRIPTION
**Pull request by victor11555**

### CODE REVIEW

File: packages/app/routes/api/comments.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  },
+  {
+    path: 'likes',
+    select: userFieldSelection
+  }
+]
+
+router.get(
+  '/:commentId',
+  passport.authenticate('jwt', { session: false }),
```
Comment (veiga91) [inline packages/app/routes/api/comments.js:48]: is this route a public one?

File: packages/app/routes/api/comments.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+      },
+      'User is getting comment replies'
+    )
+    try {
+      const comment = await Comment.findOne({
+        _id: req.params.commentId
+      }).populate(populateParams)
+      let { replies } = comment
+
+      if (req.query.sort === 'asc') {
```
Comment (veiga91) [inline packages/app/routes/api/comments.js:64]: should we be using JS to do this manipulation?

**Review by NilsonBertolaAB — APPROVED** [review]

**Review by veiga91 — APPROVED** [review]

File: packages/app/routes/api/comments.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+        replies = replies.sort((a, b) =>
+          a.likes.length > b.likes.length
+            ? 1
+            : b.likes.length > a.likes.length
+            ? -1
```
Comment (veiga91) [inline packages/app/routes/api/comments.js]: I did not get it. Why are you verifying the opposite of "a.likes.length" is greater than "b.likes.length"? if you want to exclude the equals possibility you could use >= or <=

Comment (veiga91) [reply packages/app/routes/api/comments.js]: You could use "a.likes.length - b.likes.length" as well instead of this logic.

**Review by veiga91 — CHANGES_REQUESTED** [review]
Regarding my last comment, could you review the logic? I think it could be put in a simple way.

**Review by veiga91 — APPROVED** [review]

**Pull request merged**